packages/actions/resource-scheduler/src/components/ResourceScheduler/ResourceScheduler.js

```diff
@@ -37,17 +41,45 @@ const ResourceScheduler = ({
     initialize()
   }, [initialize])
 
+  const [isCollapsed, setIsCollapsed] = useState(false)
+  const [percentageSizes, setPercentageSizes] = useState([20, 80])
+  const [minPxSizes, setMinPxSizes] = useState([325, 650])
+
+  const toggleSizes = () => {
+    if (!isCollapsed) {
+      setPercentageSizes([0, 99])
+      setMinPxSizes([0, 975])
+    } else {
+      setPercentageSizes([20, 80])
+      setMinPxSizes([325, 650])
```

Comment:
The default values for minPxSizes and percentageSizes could be extracted in a constant